### PR TITLE
driver: Clean up USB ID definitions

### PR DESCRIPTION
**Pull request by intervigilium**
Move Razer USB ID to razercommon.h and remove unnecessary #ifndef entries. The header files themselves have include guards to prevent a multiple inclusion from stomping a definition.

### CODE REVIEW

File: driver/razerkbd_driver.h [hunk 1]
```diff
@@ -13,163 +13,45 @@
 #ifndef __HID_RAZER_KEYBOARD_BLACKWIDOW_CHROMA_H
 #define __HID_RAZER_KEYBOARD_BLACKWIDOW_CHROMA_H
```
Comment (z3ntu) [inline driver/razerkbd_driver.h]: If we're at it, please change these to `__HID_RAZER_KBD_H`

**Review by z3ntu — APPROVED** [review]
LGTM, thanks! (will merge after the one comment gets addressed)

File: driver/razerkbd_driver.h (hunk 1 above)
Comment (intervigilium) [reply driver/razerkbd_driver.h]: I'll add a follow-up patch to this pull request to do that.

Comment (intervigilium) [reply driver/razerkbd_driver.h]: See second patch in this pull request.

**Comment by z3ntu** [comment]
Thanks again!

**Pull request merged**